.github/workflows/main.yml

```
@@ -0,0 +1,72 @@
+---
+name: main
+
+defaults:
+  run:
+    # In addition to GitHub's default, also abort when using undefined variables, and output individual commands.
+    # For the default, see
+    # <https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell>.
+    shell: 'bash --noprofile --norc -euo pipefail -x {0}'
+
+'on':
+  push:
+    branches:
+      - main
```

The branch is named `master`.
```suggestion
      - master
```